crates/sui-adapter-transactional-tests/tests/upgrade/dep_override.move

```diff
@@ -50,7 +50,7 @@ module Test_DepV1::DepM1 {
 
 //# upgrade --package Test_DepV1 --upgrade-capability 4,1 --dependencies Test_DepDepV2 --sender A
 module Test_DepV2::DepM1 {
-    use Test_DepDepV1::DepDepM1;
+    use Test_DepDepV2::DepDepM1;
```

Comment:
Existing tests are updated to refer to the most recent dependencies (which gets mapped to the right addresses during compilation/upgrade). Aesthetically this is nice because it corresponds to the dependency specified in `--dependencies`